config/purposes/duplex_seq.yml

```diff
@@ -0,0 +1,63 @@
+# This configuration file is for the plate and tube purposes required for the
+# high throughput Duplex-Seq pipeline
+---
+LDS Stock:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+LDS Cherrypick:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: true
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+LDS Stock XP:
+  :asset_type: plate
+LDS AL Lib:
+  :asset_type: plate
+  :label_template: plate_lds_al_lib
+  :file_links:
+  - name: 'Download Hamilton AL Lib to QC1 CSV'
+    id: 'hamilton_al_lib_to_qc1'
+  - name: 'Download Concentration (nM) CSV for Customer'
+    id: 'duplex_seq_al_lib_concentrations_for_customer'
```

Comment:
is there a reason for this one to be prefixed with 'duplex_seq' and the above one not?

Reply:
I guess makes sense if there are other 'AL Libs' that aren't for duplex seq